.github/workflows/secretlint-via-docker-only-on-changed-files.yml

```diff
@@ -0,0 +1,26 @@
+name: Secretlint via Docker only on changed files
+on: [push, pull_request]
+
+# secretlint + git diff on Pull Request
+# https://github.com/secretlint/secretlint
+
+jobs:
+  run-secretlint-via-docker:
+    name: "Secretlint via Docker only on changed files"
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: technote-space/get-diff-action@v6
+      - name: Run secretlint command on docker
+        uses: addnab/docker-run-action@v3
+        with:
+          # https://hub.docker.com/r/secretlint/secretlint/tags
+          image: secretlint/secretlint:v5.1.1
+          # https://github.com/secretlint/secretlint#using-docker
+          options: >
+            -v ${{ github.workspace }}:${{ github.workspace }}
+            -w=${{ github.workspace }}
+            --rm
+          run: secretlint ${{ env.GIT_DIFF_FILTERED }} # run only on changed files
```

Comment:
This workflow's output is as bellows;
https://github.com/gkzz/secretlint-github-actions-example/actions/runs/2043844623